### DISCUSSION TITLE
"social" list works for blocking Facebook in browsers but not in the app

### DISCUSSION THREAD

**Discussion by prl77** [StevenBlack/hosts#2025] [Q&A] [answered]
Even browsers on smartphones get blocked going to m.facebook.com but the app works fine. I assume it's a bunch of CDN networks that would have to be blocked? Perhaps by IP ranges, not just DNS names?

**Comment by StevenBlack** [comment, marked as answer]
Within their own app FB could be doing almost anything.

Based on your observation, they aren't routing DNS lookups through the hosts file.  They may be speaking to IP addresses directly.

**Comment by prl77** [comment]
Perhaps they have hard-coded IPs, who knows. But I was able to block the app by using pfBlockerNG's ASN feeds. This is basically all IP blocks used by large corporate networks. This is used by ISPs to route traffic properly but since it is publicly available, pfBlockerNG pulls them down and I created block rules based on those ASN aliases.
